scripts/globals/effects/healing.lua

```diff
@@ -4,12 +4,36 @@
 -- Activated through the /heal command
 --
 -----------------------------------
-require("scripts/globals/status")
+require("scripts/globals/keyitems")
+require("scripts/globals/quests")
 require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/zone")
 -----------------------------------
 
 function onEffectGain(target,effect)
     target:setAnimation(33)
+
+    -- Dances with Luopans
+    if target:getLocalVar("GEO_DWL_Locus_Area") == 1 and not target:setCharVar("GEO_DWL_Luopan", 1) then
```

Comment:
Probably meant a `not (target:getCharVar("GEO_DWL_Luopan") == 1)` here instead of `not target:setCharVar("GEO_DWL_Luopan", 1)` 😉

Reply:
Oh my gosh. Good catch!

I'm surprised this didn't introduce a bug when I ran through the quest.

I'll correct it and re-run through all the quest steps again to ensure it's working as expected. I'll also test this edge case again.

Reply:
Done.